Tasks/HelmDeployV0/Tests/L0.ts

```diff
@@ -1,16 +1,194 @@
+        assert(tr.stdout.indexOf('DeploymentDetailsApiResponse: {"mockKey":"mockValue"}') != -1, 'Web response for pushing metadata to evidence store');
+        assert(tr.succeeded, 'task should have succeeded');
+        done();
+    });
+
+    it('Run successfully with Helm upgrade (version 3) when chart name and release name are given', function(done: MochaDone) {        
+        const tp = path.join(__dirname, 'TestSetup.js');
+        const tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+        process.env[shared.TestEnvVars.connectionType] = shared.ConnectionTypes.KubernetesServiceConnection;
+        process.env[shared.TestEnvVars.command] = shared.Commands.upgrade;
+        process.env[shared.TestEnvVars.chartType] = shared.ChartTypes.Name;
+        process.env[shared.TestEnvVars.chartName] = shared.testChartName;
+        process.env[shared.TestEnvVars.releaseName] = shared.testReleaseName;
+        process.env[shared.TestEnvVars.failOnStderr] = 'false';
+        process.env[shared.isHelmV3] = 'true';
+        
+        tr.run();
+        assert(tr.stdout.indexOf(`Release "${shared.testReleaseName}" has been upgraded`) != -1, `Release upgraded`);
+        assert(tr.stdout.indexOf('# Source: testChartName/templates/serviceaccount.yaml') != -1, `Manifests extracted from release ${shared.testReleaseName}`);
+        assert(tr.stdout.indexOf('DeploymentDetailsApiResponse: {"mockKey":"mockValue"}') != -1, 'Web response for pushing metadata to evidence store');
+        assert(tr.succeeded, 'task should have succeeded');
+        done();
+    });
+
+    it('Run successfully with Helm init (version 2)', function(done: MochaDone) {        
+        const tp = path.join(__dirname, 'TestSetup.js');
+        const tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+        process.env[shared.TestEnvVars.connectionType] = shared.ConnectionTypes.KubernetesServiceConnection;
+        process.env[shared.TestEnvVars.command] = shared.Commands.init;
+        process.env[shared.TestEnvVars.failOnStderr] = 'false';
+
+        tr.run();
+        assert(tr.stdout.indexOf('$HELM_HOME has been configured') != -1, 'Helm init ran successfully');
+        assert(tr.succeeded, 'task should have succeeded');
+        done();
+    });
+
+    it('Helm init should fail (version 3)', function(done: MochaDone) {        
+        const tp = path.join(__dirname, 'TestSetup.js');
+        const tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+        process.env[shared.TestEnvVars.connectionType] = shared.ConnectionTypes.KubernetesServiceConnection;
+        process.env[shared.TestEnvVars.command] = shared.Commands.init;
+        process.env[shared.TestEnvVars.failOnStderr] = 'false';
+        process.env[shared.isHelmV3] = 'true';
+
+        tr.run();
+        assert(tr.failed, 'task should have failed');
```

Comment:
Add an assertion for the expected message

Reply:
`helm init` is not an expected command in Helm 3. So, there's no error message. It just prints the available commands. Should I write an assert for that?